compiler/src/compile.re

```diff
@@ -165,13 +165,7 @@ let next_state = (~is_root_file=false, {cstate_desc, cstate_filename} as cs) =>
       TypedWellFormed(typed_mod);
     | TypedWellFormed(typed_mod) =>
       Linearized(Linearize.transl_anf_module(typed_mod))
-    | Linearized(anfed) =>
-      switch (Grain_utils.Config.optimization_level^) {
-      | Level_one
-      | Level_two
-      | Level_three => Optimized(Optimize.optimize_program(anfed))
-      | Level_zero => Optimized(anfed)
-      }
+    | Linearized(anfed) => Optimized(Optimize.optimize_program(anfed))
```

Comment:
So, to make sure I understand, we have no way of disabling optimizations with this change?

Reply:
Not the Grain ones, anyway! We've flirted with the idea of removing that choice anyway because Grain programs don't work without the Grain optimizations 😅 

We may have some sort of debug mode for development (though passing compiler hooks via the command line would be a lil nicer I think).